paddle/fluid/framework/operator.cc

```diff
@@ -428,6 +429,8 @@ bool ExecutionContext::HasOutput(const std::string& name) const {
 }
 
 const Variable* ExecutionContext::InputVar(const std::string& name) const {
+  LogVarUsageIfUnusedVarCheckEnabled(name);
```

Comment:
Call this function with `if (FLAGS_enable_unused_var_check)`?

Reply:
Thanks. To reduce code duplication, I will put the if statement inside LogVarUsageIfUnusedVarCheckEnabled.